device-types/Raritan/PX2-5205R.yaml

```diff
@@ -0,0 +1,39 @@
+---
+manufacturer: Raritan
+model: PX2-5205R
+slug: px2-5205r
+part_number: PX2-5205R
+u_height: 0
+is_full_depth: false
```

Comment:
Please add the `airflow` data point if you have/can find it

Reply:
It's a PDU, no fans/no airflow that I'm aware of